src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/GenerateNuGetPackageProjectContextMenuCommand.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
 {
     [ProjectCommand(ManagedProjectSystemPackage.ManagedProjectSystemCommandSet, ManagedProjectSystemPackage.GenerateNuGetPackageProjectContextMenuCmdId)]
-    //[AppliesTo(ProjectCapability.GenerateNuGetPackage)]
+    //[AppliesTo(ProjectCapability.Pack)]
```

Comment:
These are commented out because the page isn't ready yet, right? Do we have a bug or something we can attach for tracking?